widget/embedded/src/pages/WalletsPage.tsx

```diff
@@ -11,11 +11,12 @@ import { useAppStore } from '../store/AppStore';
 import { getContainer } from '../utils/common';
 
 const ListContainer = styled('div', {
```

Comment:
I think it's good idea to have two columns in small screens but i'm not sure about padding/spacing between items. They are not ideal in my opinion.
![Screenshot from 2023-12-11 14-01-54](https://github.com/rango-exchange/rango-client/assets/99180934/303f68ee-7c1d-494a-801b-fcc723e052d9)
![Screenshot from 2023-12-11 14-04-47](https://github.com/rango-exchange/rango-client/assets/99180934/82c15cda-3dac-4219-8977-f7948dae3c9f)

Reply:
I don't have a better idea that is close to the design.